### ISSUE TITLE
Logging of application activity

### ISSUE THREAD

**Issue by kgf** [ReadyResponder/ReadyResponder#587]
We would like to have a log of system activity. This would be at a much higher level than the Rails log. It should be it's own model. Category would be what model was being changed.

timestamp datetime
user_id association into users
message string
category string

An example would be `message: "Updated  Item 6, truck", category: "Item"

**Comment by lena-pl** [comment]
I would be keen to give this issue a go! Heads up: I'm a first-time contributor to this project, so might need some patience :)

**Comment by kgf** [comment]
We welcome first time contributers ! I'll mark this as claimed. 
Are you comfortable with your first steps?

Thanks for your help. 
Kevin

**Comment by lena-pl** [comment]
@kgf I was looking at this change today. Begun implementing my logger through an after_update hook, when I noticed that you've been using the gem paper_trail in the project since da7e5efc. Is there any particular logging functionality you're looking for that paper_trail doesn't provide? Are you just looking for more readable version diffs/messages?

**Comment by kgf** [comment]
Good question. This is looking to answer the question "What happened to my data today?" Paper trail seems to answer "What happened to this object since it was created?" 
If we could implement this using a different view of the paper trail data, that would be great.
Thanks for your help !
Kevin

**Comment by lena-pl** [comment]
Proposed fix in pull request #617

**Comment by kgf** [comment]
@lena-pl Thanks for your help on this. The code had a small issue when deployed. Some of the objects, such as availabilities, do not have a last editor, as they're created from a text message.
Is this something you'd be willing to take a look at ?
Thanks
Kevin

**Comment by lena-pl** [comment]
Yeah, could do! Do you have a stack trace or something for me to look at as a starting point?

**Comment by kgf** [comment]
Sorry, I pulled this from a development stack
![screen shot 2018-05-01 at 6 04 40 pm](https://user-images.githubusercontent.com/2431417/39495876-3faf3c38-4d6a-11e8-97fd-d2dffad1cd94.png)
Thanks for your help !
